src/test/java/core/basesyntax/service/impl/WriterServiceImplTest.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertThrows;
+
+import core.basesyntax.service.WriterService;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class WriterServiceImplTest {
+    private static WriterService writerService;
+
+    @BeforeAll
+    public static void beforeAll() {
+        writerService = new WriterServiceImpl();
+    }
+
+    @Test
+    void readFile_nullPath_notOk() {
+        assertThrows(Exception.class,
+                () -> writerService.writeReport(null, "some text"));
+    }
```

Comment:
add test for positive scenario when your service will write in file, you can check it in coverage tests